tests/test_backends.py

```diff
@@ -94,9 +97,8 @@ class TestingBackendQPUs(unittest.TestCase):
     credentials.
     """
 
-    @pytest.mark.qpu
+    @pytest.mark.api
```

Comment:
The only method being tested in this class is marked as qpu. Both the setup and the test should be marked as the same since they require one another.
I believe both tests should be marked as `qpu` here.